### ISSUE TITLE
Text-classification plugin bug.

### ISSUE THREAD

**Issue by gonzalolc** [NVIDIA/DIGITS#1261]
Hello again.
I'm testing the new text classification plugin in DIGITS 5 and it is working well except for the fact that it is not plotting the validation accuracy in the graph.
I tested the same example (DBPedia) with an image dataset and it does print accuracy in the graph.
How can I fix this?

**Comment by gheinrich** [comment]
When using Torch, only image classification networks can display the accuracy during training. You can compute the accuracy yourself by testing examples. Closing as this is not a bug.

**Comment by gonzalolc** [comment]
It would be great to add an accuracy graphic. Or some tool like the confusion matrix and accuracy per class, implemented in image classification.

**Comment by gheinrich** [comment]
I agree. Feel free to contribute a change, thanks!
